pkgs/development/python-modules/pyairvisual/default.nix

```diff
@@ -46,6 +56,8 @@ buildPythonPackage rec {
     pysmb
   ];
 
+  __darwinAllowLocalNetworking = true;
```

Comment:
Is it possible to comment why this is necessary? (at least generally: ex. a test or something)

Reply:
Sounds good, I added a comment.